haproxy/values.yaml

```diff
@@ -0,0 +1,28 @@
+replicaCount: 1
+
+image:
+  repository: docker.io/haproxytech/haproxy-debian
+  tag: 2.1
+  pullPolicy: IfNotPresent
+
+ingress:
+  enabled: false
+
+service:
+  port: 8080
+  type: ClusterIP
+
+haproxy:
+  config: tls
+  frontend:
+    port: 30636
+  backend: example.org
+  certificate:
+    path: /usr/share/ca-certificates/mozilla/COMODO_RSA_Certification_Authority.crt
```

Comment:
Would change the path to `/etc/ssl/certs/ca-certificates.crt`. All the CAs are included there.